management_api_app/services/authentication.py

```diff
@@ -0,0 +1,120 @@
+from fastapi import Request, Depends, HTTPException, status
+from fastapi.security import OAuth2AuthorizationCodeBearer, HTTPBearer
+from typing import List, Optional
+from core import config
+import base64
+import jwt
+import requests
+import rsa
+
+class User:
```

Comment:
Also is User a wrapper over a dict? Why not use User(dict) as delegate everything to it?